clouddrift/sphere.py

```diff
@@ -660,13 +664,12 @@ def cartesian_to_spherical(
     return lon, lat
 
 
+T = TypeVar("T", bound=float | np.ndarray)
```

Comment:
I am not finding this alias very readable. Why does this help?

Reply:
This isn't an alias, its a generic type variable: https://mypy.readthedocs.io/en/stable/generics.html

Reply:
Sorry, the whole typing thing is confusing to me

Reply:
I read more about `TypeVar` and actually like it. If I understand correctly, it assigns the type to `T` dynamically, corrct?